File: app/src/main/java/com/rafamatias/nlp/presentation/view/activity/MainActivity.java
package com.rafamatias.nlp.presentation.view.activity;

import android.databinding.DataBindingUtil;
import android.support.v4.view.ViewPager;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.rafamatias.nlp.R;
import com.rafamatias.nlp.databinding.ActivityMainBinding;
import com.rafamatias.nlp.presentation.view.adapter.TweetViewPagerAdapter;
import com.rafamatias.nlp.presentation.view.fragment.TweetFragment;

/**
 * A simple {@link AppCompatActivity} subclass.
 * Uses {@link ViewPager} to show fragments of {@link TweetFragment}
 * @see TweetViewPagerAdapter
 * @see TweetFragment
 */
public class MainActivity extends AppCompatActivity {

    private ActivityMainBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = DataBindingUtil.setContentView(this, R.layout.activity_main);

        setupViewPager();
    }

    private void setupViewPager() {
        TweetViewPagerAdapter viewPagerAdapter = new TweetViewPagerAdapter(getSupportFragmentManager());
        binding.viewPager.setAdapter(viewPagerAdapter);
    }
}

File: app/src/main/java/br/com/andreyneto/ifood_mobile_test/MainActivity.kt
package br.com.andreyneto.ifood_mobile_test

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.WindowManager

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        window.setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_STATE_ALWAYS_VISIBLE)
        window.setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_ADJUST_RESIZE)

    }
}
